src/agl/web_app_manager_service_agl.cc

```diff
@@ -341,6 +341,8 @@ void WebAppManagerServiceAGL::LaunchStartupAppFromJsonConfig() {
     LOG_DEBUG("Failed to parse %s configuration file", configfile.c_str());
   }
 
+  root["folderPath"] = startup_app_uri_.c_str();
```

Comment:
Well, we shouldn't need it if provided by json file. So I guess we're not going to right?

Reply:
Come to think of, probably this is much better, as we're already specifying that in the dir name, just that seems weird we basically create a different version of the json file. We basically do the same with the witdhOverride/heightOverride values, just that those seemed a bit harder to remove or adapt, so I left it as is. 

This seems fine w/ me, if we want to have this way.

Reply:
We already need the --app-install-dir to find appinfo.json. So having a folderPath in the appinfo.json only makes sense if we want the folder to be different from the appinfo.json. It makes sense in certain cases, usually with a relative path (i.e. having app in /usr/lib/wam_apps/app_name/appinfo.json, but then the html files in /usr/lib/wam_apps/app_name/src/index.html).

For now we do not need that, but in the future we could support folderPath with relative paths that way. I don't think we are interested in having appinfo.json in a path that is not the actual application path usually.